### PR TITLE
P1/refactor/90 user context

### PR DESCRIPTION
**Pull request by netlightebba**

### CODE REVIEW

**Review by netlight-saraandersson — APPROVED** [review]
Awsome :D

**Comment by netlight-johannadagfalk** [comment]
Perhaps wait for three approvals before merging this one so that we all can have a look to understand :D

**Comment by netlight-johannadagfalk** [comment]
> Perhaps wait for three approvals before merging this one so that we all can have a look to understand :D

I´ve started but is not finished!

**Review by netlight-johannadagfalk — APPROVED** [review]
Looks good :D

File: app/assets/main/react_components/helpers/AchievementsHelper.js
Comment (netlightebba) [reply app/assets/main/react_components/helpers/AchievementsHelper.js:49]: Haha, I will fix this! 🙈

File: app/assets/main/react_components/helpers/AchievementsHelper.js
Comment (elinaolsson) [inline app/assets/main/react_components/helpers/AchievementsHelper.js:83]: Is this function only for Drag or is it for Move like one of the previous functions?

File: app/assets/main/react_components/helpers/AchievementsHelper.js
Comment (elinaolsson) [inline app/assets/main/react_components/helpers/AchievementsHelper.js:172]: Maybe a more clear name of what is combined, e.g. "combinedActionsArray"?

File: app/assets/main/react_components/helpers/AchievementsHelper.js
Comment (elinaolsson) [inline app/assets/main/react_components/helpers/AchievementsHelper.js:175]: Same here, what is combined?

File: app/assets/main/react_components/helpers/AchievementsHelper.js
Comment (elinaolsson) [inline app/assets/main/react_components/helpers/AchievementsHelper.js:178]: Might also be clarified, what result?

File: app/assets/main/react_components/helpers/UserActionsHelper.js
Comment (elinaolsson) [inline app/assets/main/react_components/helpers/UserActionsHelper.js:16]: Is "done" a word we use on other places? Or is "performed" or "completed" better?

**Review by elinaolsson — APPROVED** [review]
Wow grymt jobbat! 🤩  🥇

**Pull request merged**
